addon/components/ember-collection.js

```diff
@@ -137,26 +138,26 @@ export default Ember.Component.extend({
   // updateOffset() {
   //   if (this.element) {
   //     console.log('scroll');
-  //     this.offsetX = this.element.scrollLeft;
-  //     this.offsetY = this.element.scrollTop;
+  //     this._offsetX = this.element.scrollLeft;
+  //     this._offsetY = this.element.scrollTop;
   //     this.rerender();
   //   }
   // },
-  willRender() {
```

Comment:
this change makes it so the items are never updated after initial render. The `rAF` callback just calls `rerender` but this won't trigger the `updateCells` code path. Instead of calling `rerender` should we just do a `Ember.run.scheduleOnce(this, 'updateCells')` in the `rAF` callback?